httpd/handler.go

```diff
@@ -174,11 +179,110 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 		return
 	}
 
-	// Execute query. One result will return for each statement.
-	results := h.server.ExecuteQuery(query, db, user)
+	// get the chunking settings
+	chunked := q.Get("chunked") == "true"
+	// even if we're not chunking, the engine will chunk at this size and then the handler will combine results
+	chunkSize := DefaultChunkSize
+	if chunked {
+		if cs, err := strconv.ParseInt(q.Get("chunk_size"), 10, 64); err == nil {
+			chunkSize = int(cs)
```

Comment:
This can overflow on 32-bit machines. If we don't want to support chunking of sizes more than 32-bits, we should just call `ParseInt`, passing in 32-bits as the maximum width.